pkg/apis/pulumi/v1alpha1/stack_types.go

```diff
@@ -29,14 +29,14 @@ type StackSpec struct {
 	// Deprecated: use EnvRefs instead.
 	SecretEnvs []string `json:"envSecrets,omitempty"`
 
-	// (optional) Backend is an optional backend URL to use for all Pulumi operations.
-	// Examples:
-	//   - Pulumi Service:              "https://app.pulumi.com" (default)
-	//   - Self-managed Pulumi Service: "https://pulumi.acmecorp.com"
-	//   - Local:                       "file://./einstein"
-	//   - AWS:                         "s3://<my-pulumi-state-bucket>"
-	//   - Azure:                       "azblob://<my-pulumi-state-bucket>"
-	//   - GCP:                         "gs://<my-pulumi-state-bucket>"
+	// (optional) Backend is an optional backend URL to use for all Pulumi operations.<br/>
```

Comment:
It seems odd to include HTML snippets in Go comments

Reply:
Yeah consider this to just be a stop-gap for now... I am still investigating a better alternative for the documentation generation. In this instance, the rendering is busted without the embedded line breaks.